src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Search.cs

```diff
@@ -0,0 +1,263 @@
+            {
+                SuggestDelegationHint(node, binding);
+                return false;
+            }
+
+            return true;
+        }
+
+        public override bool UpdateDataQuerySelects(CallNode callNode, TexlBinding binding, DataSourceToQueryOptionsMap dataSourceToQueryOptionsMap)
+        {
+            Contracts.AssertValue(callNode);
+            Contracts.AssertValue(binding);
+
+            // Ignore delegation warning
+            if (!CheckArgsCount(callNode, binding, DocumentErrorSeverity.Moderate))
+            {
+                return false;
+            }
+
+            var args = Contracts.VerifyValue(callNode.Args.Children);
+            DType dsType = binding.GetType(args[0]);
+            if (dsType.AssociatedDataSources == null)
+            {
+                return false;
+            }
+
+            int cargs = args.Count;
+            bool retval = false;
+
+            for (int i = 2; i < cargs; i++)
+            {
+                string columnName;
+                DType columnType;
+                if (binding.Features.SupportColumnNamesAsIdentifiers)
+                {
+                    columnName = ((FirstNameNode)args[i]).Ident.Name;
+                    columnType = dsType.GetType(new DName(columnName));
+                }
+                else
+                {
+                    columnType = binding.GetType(args[i]);
+                    StrLitNode columnNode = args[i].AsStrLit();
+                    if (columnType.Kind != DKind.String || columnNode == null)
+                    {
+                        continue;
+                    }
+
+                    columnName = columnNode.Value;
+                }
+
+                Contracts.Assert(dsType.Contains(new DName(columnName)));
+
+                retval |= dsType.AssociateDataSourcesToSelect(dataSourceToQueryOptionsMap, columnName, columnType, true);
+            }
+
+            return retval;
+        }
+
+        public override bool IsServerDelegatable(CallNode callNode, TexlBinding binding)
+        {
+            Contracts.AssertValue(callNode);
+            Contracts.AssertValue(binding);
+
+            if (!CheckArgsCount(callNode, binding)) 
+            { 
+                return false;
+            }
+
+            if (!TryGetValidDataSourceForDelegation(callNode, binding, DelegationCapability.Filter, out var dataSource))
+            {
+                return false;
+            }
+
+            var args = Contracts.VerifyValue(callNode.Args.Children);
+            int cargs = args.Count;
+
+            FilterOpMetadata metadata = Contracts.VerifyValue(dataSource.DelegationMetadata.FilterDelegationMetadata);
+            for (int i = 2; i < cargs; i++)
+            {
+                if (!IsValidSearchableColumnNode(args[i], binding, dataSource))
+                {
+#pragma warning disable CA1305 // Specify IFormatProvider
+                    var message = string.Format("ColumnSearchable: false");
```

Comment:
>string.Format("ColumnSearchable: false") [](http://example.com/codeflow?start=34&length=40)

This is the same as `"ColumnSearchable: false"` so there's no need to use string.Format (and then no need to disable/restore the warning)